boards/arduino-uno/Makefile.features

```diff
@@ -0,0 +1,10 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+# Various other features (if any)
+
```

Comment:
maybe you can try to add:
```
FEATURES_PROVIDED += arduino
FEATURES_PROVIDED += cpp
```
to enable arduino sketches via RIOT on arduino-uno board ;)